CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 # See https://help.github.com/articles/about-code-owners/
 
 ### General infra ###
-/* @dotnet/dotnet-docker-reviewers
+* @dotnet/dotnet-docker-reviewers
 eng/ @dotnet/dotnet-docker-reviewers

```

Comment:
Given the previous line, I don't think this line is necessary and adds maintenance overhead.

Reply:
Ah thanks - I overlooked this.